frontend/src/metabase/static-viz/components/LineAreaBarChart/LineAreaBarChart.tsx

```diff
@@ -60,9 +66,34 @@ const LineAreaBarChart = ({
     chartSize,
   );
 
+  const keys = series
+    .map(singleSeries => {
+      if (singleSeries.seriesKey) {
+        return singleSeries.seriesKey;
+      }
+
+      return formatStaticValue(singleSeries.name, {
+        column: singleSeries.column,
+      });
+    })
+    .filter(isNotNull);
+  const palette = { ...colors, ...instanceColors };
+  const seriesColors = settings.series_settings
```

Comment:
This achieve the same thing as 
https://github.com/metabase/metabase/blob/cc589cf0f6a3f4dfe8e22aee2c1f21a8a595ddd0/frontend/src/metabase/visualizations/lib/settings/series.js#L178-L182

but with simpler logic.